docs/getting-started/quickstart.md

```diff
@@ -39,49 +43,69 @@ The [`run`](https://substreams.streamingfast.io/reference-and-specs/command-line
 **Tip**: Try the [Python](https://github.com/streamingfast/substreams-playground/tree/master/consumers/python) example if you prefer streaming by using third-party languages.
 {% endhint %}
 
-## Platform independent Substreams
+## Build a Substreams module
 
-Substreams is platform independent, meaning you can use many different blockchains.&#x20;
+To create a simple "Substreams module" that extracts data from the Ethereum blockchain, you will need to use the [`substreams` CLI](https://substreams.streamingfast.io/reference-and-specs/command-line-interface) and obtain an authentication key.
 
-Developers select a specific blockchain and build a Substreams module tailored to the selected chain.&#x20;
+Before continuing, ensure that your system [meets the basic requirements](https://substreams.streamingfast.io/developers-guide/installation-requirements) for Substreams development.
 
-Data is available for any blockchain exposing an operational Firehose endpoint, installed and set up in [an on-premises environment](https://firehose.streamingfast.io/firehose-setup/ethereum/installation-1), or [provided by StreamingFast](../reference-and-specs/chains-and-endpoints.md) or other vendors.
+### Objectives
 
-{% hint style="info" %}
-**Note**: The remaining documentation assumes the [`substreams` CLI](../reference-and-specs/command-line-interface.md) and all other required dependencies have been installed and a [StreamingFast authentication token](../reference-and-specs/authentication.md) has been obtained.&#x20;
-{% endhint %}
+- Create your first Substreams module
+- Use the [`substreams` CLI](https://substreams.streamingfast.io/reference-and-specs/command-line-interface) to run the module
 
-### **Basics**
+### Prerequisites
 
-The most basic approach to use Substreams is through the [`substreams` CLI](../reference-and-specs/command-line-interface.md), passing an endpoint and the name of the Rust function, or module handler, used by the compute engine for processing.&#x20;
+- Go compiler
+- Rust compiler
+- [`substreams` CLI](https://substreams.streamingfast.io/reference-and-specs/command-line-interface)
+- StreamingFast authentication key
```

Comment:
Most of this should have been done by that point already. For example, there is `Run your first Substreams` higher, so I pretty sure that authentication and Substreams CLI has been dealt with.

The Go compiler, is definitely not required, what made you think it's required?

I would get rid of the `Prerequisites` section and just ensure prior installation step are covered.

Reply:
Removed. Was matching the Google QSG here.